lib/cli/test/run_tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
This runs tests?
I don't understand what's going on here.

Can't we write jest tests and run them in jest?

Reply:
If this needs to be this way, we need a good documentation on what this code does, either in a `readme.md` or above code.

Reply:
Yes we can, but it's much easier to do those manipulations in bash then in js, plus it's straightforward to reproduce particular steps: you just copypaste them to your console. My inspiration were CRA's e2e tests: https://github.com/facebookincubator/create-react-app/tree/master/tasks

Reply:
I have added a section to `CONTRIBUTING.md`, and some inline comments